apps/server/lib/lexical/server/code_intelligence/completion.ex

```diff
@@ -69,6 +79,18 @@ defmodule Lexical.Server.CodeIntelligence.Completion do
     end
   end
 
+  defp prefix_not_a_trigger?(env) do
```

Comment:
`prefix_is_trigger?` and reverse the logic in the function. Then `not prefix_is_trigger?` in the cond.

Always name functions and variables after the positive case.

Reply:
Good habits.